frontend/editor.css

```diff
@@ -3320,22 +3320,56 @@ pluto-cell.hooked_up pluto-output {
 .pluto-frontmatter {
     font-family: var(--system-ui-font-stack);
     width: min(31rem, 90vw);
-    color: var(--black);
-    background: var(--white);
+    color: var(--export-color);
+    background: var(--export-bg-color);
+    border-radius: 1em;
 }
 
 .pluto-frontmatter button {
     cursor: pointer;
+    background-color: var(--frontmatter-button-bg-color);
+    border-radius: 0.5em;
+    border: 2px solid var(--frontmatter-button-bg-color);
+    padding: 0.3em 0.5em;
+    font-weight: 500;
+}
+
+.pluto-frontmatter button:hover {
+    border-color: var(--frontmatter-input-border-color);
+}
+
+.pluto-frontmatter input {
+    background-color: var(--frontmatter-input-bg-color);
+    border-radius: 0.5em;
+    border: 2px solid var(--frontmatter-input-border-color);
+    padding: 0.3em 0.5em;
+}
+
+.pluto-frontmatter input:focus {
+    outline: none;
+}
+
+.pluto-frontmatter label {
+    text-transform: capitalize;
+    font-weight: 500;
 }
 
 .pluto-frontmatter .deletefield {
```

Comment:
The buttons are a bit too small here (Chrome). Could you make them more visible?

<img width="457" alt="Scherm­afbeelding 2023-02-07 om 19 46 43" src="https://user-images.githubusercontent.com/6933510/217337873-ff6dc050-797b-4f31-9fe1-5626368f5037.png">

Reply:
Could you also make sure that they are visible in light mode?

<img width="460" alt="Scherm­afbeelding 2023-02-07 om 19 48 06" src="https://user-images.githubusercontent.com/6933510/217338093-1a2be075-595b-45eb-ae3f-9af4ad31ae58.png">

Reply:
I've made the buttons bigger, and made sure they inherit the correct text color :slightly_smiling_face:. This is what I see now (in chrome):

![image](https://user-images.githubusercontent.com/20903656/222157875-d7dd5307-4eb9-41b7-a330-e7d7483604cc.png)